src/main/java/com/aws/greengrass/logmanager/LogManagerService.java

```diff
@@ -609,44 +608,42 @@ private void processLogsAndUpload() throws InterruptedException {
                         continue;
                     }
 
-                    ComponentLogFileInformation unitOfWork = ComponentLogFileInformation.builder()
+                    ComponentLogFileInformation logFileInfo = ComponentLogFileInformation.builder()
                             .name(componentName)
                             .multiLineStartPattern(componentLogConfiguration.getMultiLineStartPattern())
                             .desiredLogLevel(componentLogConfiguration.getMinimumLogLevel())
                             .componentType(componentLogConfiguration.getComponentType())
                             .logFileGroup(logFileGroup)
                             .build();
 
-                    unitsOfWork.add(unitOfWork);
+                    componentMetadata.add(logFileInfo);
 
                     logFileGroup.forEach(file -> {
                         long startPosition = 0;
                         String fileHash = file.hashString();
-                        // The file must contain enough lines for digest hash, otherwise fileHash is empty string
-                        if (!HASH_VALUE_OF_EMPTY_STRING.equals(fileHash)) {
```

Comment:
This red block is just removing this if statement. The log group will filter out files that are no ready to be processed